vlib/time/README.md

```diff
@@ -1,17 +1,117 @@
+# time
+
+A time library
+
+## Features
+
+TODO
+
 ## Description:
 
 `time` provides utilities for working with time and dates:
+
 - parsing of time values expressed in one of the commonly used standard time/date formats
 - formatting of time values
 - arithmetic over times/durations
 - converting between local time and UTC (timezone support)
 - stop watches for accurately measuring time durations
 - sleeping for a period of time
 
-## Examples:
+**Example:**
```

Comment:
Why a singular `Example`, instead of the plural `Examples` ?
The original was better, given that what follows are many independent code sections, not just one.